### ISSUE TITLE
Viewport issue on iphone 4 inches

### ISSUE THREAD

**Issue by kirkas** [phonegap-build/StatusBarPlugin#19]
Viewport is wrong on iphone 4 inche since i've added this plugin. As you can see the window height return 656px instead to the correct 568px.

![screen shot 2014-02-14 at 08 57 27](https://f.cloud.github.com/assets/1159783/2172872/3f36835c-9599-11e3-8d7d-561824d32f4d.png)

**Comment by billstremor** [comment]
I'm having the exact same issue with the plugin. viewport is reporting 656 tall.

**Comment by billstremor** [comment]
Is there a fix or viable workaround for this?

**Comment by kirkas** [comment]
Changing the viewport meta to:

```
<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height" />
```

work for me

**Comment by orenagiv** [comment]
I can confirm that using "height=device-height" does solve the issue, but only AFTER Cordova is ready.
And there's still the bug that if you start the App in Landscape mode - then half the screen is white (at least while using PGB 3.3):
https://github.com/phonegap-build/StatusBarPlugin/issues/4

Hope that will be solved soon.

**Comment by wildabeast** [comment]
Try the latest version (1.1.0) of this plugin -- it should fix this issue.

**Comment by orenagiv** [comment]
Awesome!!!
I can confirm it's working perfectly now :)

Many thanks!

**Comment by wildabeast** [comment]
Cool thanks. Closing.

**Comment by orenagiv** [comment]
Ehh... Sorry about that.. but I noticed something new which might be related:
If you show the status-bar and the hide it - the screen size increases in around 20 pixels in width and height.
If you show and hide it again - it will increase in another 20 pixels.

Shall I open a separate issue for this one?

**Comment by wildabeast** [comment]
Ugh yes please open a new one.

**Comment by orenagiv** [comment]
Opened:
https://github.com/phonegap-build/StatusBarPlugin/issues/21
